mariaDB4j-core/src/main/java/ch/vorburger/mariadb4j/DB.java

```diff
@@ -99,7 +99,7 @@ public static DB newEmbeddedDB(int port) throws ManagedProcessException {
         return newEmbeddedDB(config.build());
     }
 
-    ManagedProcess installPreparation() throws ManagedProcessException, IOException {
+    protected ManagedProcess createMysqlInstallProcess() throws ManagedProcessException, IOException {
```

Comment:
I'm trying to avoid any MySQL vs MariaDB naming, especially in the API, how about just:

```suggestion
    protected ManagedProcess createDBInstallProcess() throws ManagedProcessException, IOException {
```

Reply:
Good call.